source/static/whatsNew/2.0.1.json

```
@@ -0,0 +1,34 @@
+[
+  {
+    "description": "Fix migration crash",
```

```suggestion
    "description": "Fix startup crash",
```

I updated my PR to be more clear